analytics.js

```diff
@@ -180,12 +182,26 @@ class Analytics {
         this.loadOnlyIntegrations,
         this.clientIntegrations
       );
+      let cookieConsent = undefined;
+      // Check if cookie consent manager is being set through load options
+      if (this.cookieConsentOptions) {
+        // Call the cookie consent factory to initialise and return the type of cookie
```

Comment:
Found one issue with this checking. Will resolve and raise a PR to production-staging.